src/app/containers/InlineLink/index.jsx

```diff
@@ -31,9 +33,16 @@ const InlineLinkContainer = ({ locator, blocks }) => {
 
   // else return a normal hyperlink
   return (
-    <InlineLink href={locator}>
-      <Blocks blocks={blocks} componentsToRender={componentsToRender} />
-    </InlineLink>
+    <ServiceContextConsumer>
+      {({ externalLinkText }) => (
+        <InlineLink href={locator}>
+          <Blocks blocks={blocks} componentsToRender={componentsToRender} />
+          {isExternal ? (
+            <VisuallyHiddenText>{externalLinkText}</VisuallyHiddenText>
+          ) : null}
```

Comment:
Based on Psammead meeting today - `externalLinkText` will be a prefix on Persian and a suffix on News, right? 🤔

So we've i18ned the text here but not the placement of it 😅

Reply:
Yes, agreed. We have several examples of containers with translated text. In Brand, Caption, Copyright, ErrorMain. 
`brandName`
`imageCaptionOffscreenText`
`imageCopyrightOffscreenText`